tools/autograd/gen_variable_type.py

```diff
@@ -646,20 +651,36 @@ def save_variables(
                 stmts.append('}')
         return stmts
 
-    def emit_dispatch_call(f: NativeFunction, input_base: str, unpacked_args: Sequence[str]) -> str:
+    def emit_dispatch_call(f: NativeFunction, input_base: str, unpacked_args: Sequence[str], *, is_view_call: bool) -> str:
```

Comment:
Could you add a block comment at the beginning of this function to explain if/when we redispatch, what are the criterions and why we do that please? That would make this function a bit easier to understand in the future.

Reply:
agreed, will do!